src/flop.sol

```diff
@@ -50,16 +50,16 @@ contract Flopper is LibNote {
 
     // --- Data ---
     struct Bid {
-        uint256 bid;
-        uint256 lot;
+        uint256 bid;  // dai paid      [rad]
+        uint256 lot;  // gems for sale [wad]
         address guy;  // high bidder
         uint48  tic;  // expiry time
```

Comment:
`tic` & `end` should have comments similar to those suggested for the `Flipper`.